docs/docs/data-sources/Intents.md

```diff
@@ -0,0 +1,60 @@
+
+**Questions** are the queries that users will ask your chatbot. They can be phrased in a variety of ways, and the system will attempt to match them to the appropriate intent. For example, if you were creating an intent for a chatbot that answers questions about algorithms, you might include questions such as:
+
+- What is an algorithm?
+- What is a sorting algorithm?
+- What is a binary search algorithm?
+
+**Answers** are the responses that your chatbot will provide when a user asks a question. They can be simple text responses or more complex, such as a link to a website or a PDF document. For example, if you were creating an intent for a chatbot that answers questions about the weather, you might include answers such as:
+
+- An algorithm is a set of instructions for solving a problem or performing a task.
+- A sorting algorithm is an algorithm that puts elements of a list in a certain order.
+- A binary search algorithm is an algorithm that finds the position of a target value within a sorted array.
+
+Adding these intents to your chatbot will allow it to answer questions about algorithms. You can also add more complex answers, such as links to websites or PDF documents.
+
+### Intent fields
+
+An intent has 4 main fields
+
+| Field Name            | Description                                                                                                      | Example                                                                                                                                                                                       | Mandatory |
+| --------------------- | ---------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------- |
+| Name                  | This is the name of the intent                                                                                   | _Pricing_                                                                                                                                                                                     | Yes       |
+| Question              | This is a visitor-friendly way for the intent. This is very useful if we want to display this intent in a button | _What is the price_                                                                                                                                                                           | Yes       |
+| Alternative phrase(s) | \- These are in fact training data that help the chatbot map similar questions to the same intent\.              | _Is Algomo expensive?_                                                                                                                                                                        | No        |
```

Comment:
The descriptions here aren't the best. Not sure exactly how to improve it though. Maybe something like this? (i got this from ChatGPT)

Intent Name: Intent names are labels used to identify specific user topics or queries, helping in organization and categorization.
Main Question of an Intent: It represents the primary query or task a user wants to achieve during the interaction.
Alternative Phrases/Utterances of an Intent: These are variations of how users express the same intent, improving the chatbot's understanding of diverse user inputs.
Response of an Intent: It's the chatbot's informative reply triggered when the intent is recognized, addressing the user's query or request.